Functions.Templates/Bindings/bindings.json

```diff
@@ -2278,590 +2286,6 @@
           "required": true,
           "label": "$oneDrive_out_name_label",
           "help": "$oneDrive_out_name_help"
-        },
```

Comment:
added

---
In reply to: [133859511](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859511) [](ancestors = 133859511)